### DISCUSSION TITLE
Auto Load Cfg

### DISCUSSION THREAD

**Discussion by PUREHAX** [danielkrupinski/Osiris#2627] [Q&A] [answered]
I would like to get the config loaded when I inject the cheat. How can I do this ?
I've tried hardcoding it but this is not a good option.

I'm not into c++ so if someone could do a step by step instruction I would be very thankful 🙏

**Comment by linuxer9** [comment, marked as answer]
rename your config file to default.json, it will be loaded automatically when you inject.

**Reply by PUREHAX** [reply]
Thanks! It worked

**Reply by PUREHAX** [reply]
I have one more question. Is it possilbe to not have the menu open, when injected ?

**Reply by linuxer9** [reply]
yes, but I can't test it now, 
edit GUI.h and find this line
`    bool open = true;`
change it to:
`    bool open = false;`

**Reply by PUREHAX** [reply]
Alright. Thanks :)
